### PR TITLE
Expose as ES Module

### PR DESCRIPTION
**Pull request by ukstv**
And make CRA not complain about dynamic dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,16 +4,21 @@
   "description": "Blockchain utils for 3ID",
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
+  "module": "esm/index.js",
   "scripts": {
     "test": "./node_modules/.bin/jest --coverage",
-    "build": "./node_modules/.bin/tsc -p tsconfig.json",
+    "build": "./node_modules/.bin/tsc -p tsconfig.json && ./node_modules/.bin/tsc -p tsconfig.esm.json",
```
Comment (oed) [inline package.json:10]:
```suggestion
    "build": "npm run build:es5; npm run build:esm",
    "build:es5": "./node_modules/.bin/tsc -p tsconfig.json",
    "build:esm": "./node_modules/.bin/tsc -p tsconfig.esm.json",
```

**Review by oed — COMMENTED** [review]
Looks good, just one small change I'd make.

**Comment by ukstv** [comment]
Well, @oed This pull request was premature. Thank you for not merging this.

CRA is widely used. It does not support WASM out of the box. Instead of warnings, the pull request would require to [reconfigure every CRA application](https://www.telerik.com/blogs/using-webassembly-with-react) that happens to use 3id-connect.

There are three pull requests to CRA regarding WASM, either closed or still open for a year. It is a choice between bad and worse now: make consumers reconfigure CRA vs warning (and optionally set CI to not true). I feel like warnings as in https://github.com/3box/3id-connect/issues/86 is the lesser evil, so this PR better be closed.

**Comment by ukstv** [comment]
3id-connect depends on ethereum only, so one idea is to split this into subpackages, so that one could import only what is required. Or other way to require only ethereum-related functions.

**Comment by oed** [comment]
Yeah, sub packages might be the way to go. We need to make sure that even when you pull in all packages the build is still small (which is hard regardless).

**Pull request closed without merging**